.travis.yml

```diff
@@ -6,3 +6,6 @@ rvm:
 env:
   global:
     - NOKOGIRI_USE_SYSTEM_LIBRARIES=true
+addons:
+  code_climate:
+    repo_token:
```

Comment:
[Setting Up Test Coverage](http://docs.codeclimate.com/article/219-setting-up-test-coverage) in Code Climate.